.travis.yml

```diff
@@ -2,45 +2,40 @@ language: python
 sudo: false
 dist: trusty
 cache: pip
-python:
-  - "pypy-5.4.1"
-  - "3.6"
-  - "3.5"
-  - "3.4"
-  - "2.7"
-  - "nightly"
+
 env:
   global:
-    - TEST='-v --durations 25'
     - PYTHONFAULTHANDLER=x
     - PYTHONWARNINGS=all
     - SKIP_LATEX_BUILD=1
-  matrix:
-    - DOCUTILS=0.13.1
-    - DOCUTILS=0.14
+
 matrix:
-  exclude:
-    - python: "3.4"
-      env: DOCUTILS=0.13.1
-    - python: "3.5"
-      env: DOCUTILS=0.13.1
-    - python: "3.6"
-      env: DOCUTILS=0.13.1
-    - python: nightly
-      env: DOCUTILS=0.13.1
-    - python: "pypy-5.4.1"
-      env: DOCUTILS=0.13.1
+  include:
+    - python: '3.6'
+      env: TOXENV=mypy
+    - python: '2.7'
+      env: TOXENV=py27-du13
+    - python: '2.7'
+      env: TOXENV=flake8,py27-du14
+    - python: '3.4'
+      env: TOXENV=flake8,py34-du14
```

Comment:
It seems tox sets up environment on each target.
For example, this line creates two environment.
I'm worry about it wastes time.
https://travis-ci.org/sphinx-doc/sphinx/jobs/318589078

Reply:
I can accept to separate flake8 to independent task on travis like mypy.

Reply:
Done